diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 64e4ac51952410d6fe5e5ec91981bfdb928c0a56..a79a73aef106499470e7bf95bc7eba6f41470c57 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -346,6 +346,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
