diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index c6dec5e435fee2812bc6e49570735931f935cfdd..b8cd94a623dd177853be2e2ba291c360a2b6aa3d 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1574,6 +1574,7 @@ source_set("extensions") {
       "//components/enterprise/data_controls/core/browser",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 proto_library("cws_item_service_proto") {
