diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 2837fb0bc4962c669ad009b9e44fb64287235f3a..aa886f058527c53ede5c421891b2ce2282097f20 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4118,6 +4118,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
