diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 8d085b7fb399b4d0121e054408b6e0afce973f7c..88c652245641ccfe51df88fdb2f9c516393e76f6 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2506,6 +2506,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
