diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
index ab225cf026bdaa5985fe87938dcb8fe25ab364f5..1b68d103d4793d34d2cb26f38b4b34bf73103014 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
@@ -352,7 +352,7 @@ public class ToolbarManager
     }
 
     /** An {@link ObservableSupplier<Integer>} for the browser constraints of the current tab. */
-    private static class ConstraintsProxy extends ObservableSupplierImpl<Integer>
+    public static class ConstraintsProxy extends ObservableSupplierImpl<Integer>
             implements Callback<Integer> {
         private ObservableSupplier<Integer> mCurrentConstraintDelegate;
 
