diff --git a/chrome/browser/first_run/upgrade_util_win.cc b/chrome/browser/first_run/upgrade_util_win.cc
index 3a2436151de485473127aa8f5f0b2f481b421c6e..90dbfd5dfbf3a70f9a54c53d7cc29df7259a20a8 100644
--- a/chrome/browser/first_run/upgrade_util_win.cc
+++ b/chrome/browser/first_run/upgrade_util_win.cc
@@ -57,7 +57,7 @@
 #include "third_party/abseil-cpp/absl/cleanup/cleanup.h"
 #include "ui/base/ui_base_switches.h"
 
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
 #include "chrome/updater/app/server/win/updater_legacy_idl.h"
 #endif
 
@@ -70,7 +70,7 @@ bool GetNewerChromeFile(base::FilePath* path) {
   return true;
 }
 
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
 // Holds the result of the IPC to CoCreate the process launcher.
 struct CreateProcessLauncherResult
     : public base::RefCountedThreadSafe<CreateProcessLauncherResult> {
@@ -201,7 +201,7 @@ Microsoft::WRL::ComPtr<IUnknown> CreateProcessLauncher() {
 #endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
 
 bool InvokeGoogleUpdateForRename() {
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
   // This has been identified as very slow on some startups. Detailed trace
   // events below try to shine a light on each steps. crbug.com/1252004
   TRACE_EVENT0("startup", "upgrade_util::InvokeGoogleUpdateForRename");
