diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 5803edb96805402e8587f0b1ccae8e3dcc80a605..f70a769934d0c7902a66b8c599b3d29bd184aa55 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2799,6 +2799,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
