diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 0a66362c3e3799806d3182ee82c6a0e1573085db..76248e7fd4cbae1703ed1e8e70e00b4858365d70 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -564,6 +564,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
