diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index e83da2b8634456888f6edc1f9a08e59fd8ff06b3..ad5461f09c39fc1d35441351622a3b5d3cbef829 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -243,6 +243,7 @@ class BrowserProcessImpl : public BrowserProcess,
   GlobalFeatures* GetFeatures() override;
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
