diff --git a/chrome/android/features/tab_ui/BUILD.gn b/chrome/android/features/tab_ui/BUILD.gn
index eb0e3f1b8143ef7999f87dde0136e2bf8d165b1c..da8c3c33137638c433c9e3a55700f5c8e9743db1 100644
--- a/chrome/android/features/tab_ui/BUILD.gn
+++ b/chrome/android/features/tab_ui/BUILD.gn
@@ -105,6 +105,7 @@ android_resources("java_resources") {
     "java/res/xml/tab_archive_time_delta_preference.xml",
     "java/res/xml/tabs_settings.xml",
   ]
+  sources += brave_android_features_java_resources
   deps = [
     ":java_strings_grd",
     "//chrome/android:chrome_app_java_resources",
