diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
index 90ec2f06f6e0873909a265c7bfa601ff22ecebdd..72289184882fe24b636c5f805f8b1719ed8b2788 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
@@ -1250,7 +1250,7 @@ public class TabbedRootUiCoordinator extends RootUiCoordinator {
         mNotificationPermissionController =
                 new NotificationPermissionController(mWindowAndroid, rationaleUIDelegateSupplier);
         NotificationPermissionController.attach(mWindowAndroid, mNotificationPermissionController);
-        if (mNotificationPermissionController.requestPermissionIfNeeded(/* contextual= */ false)) {
+        if (false && mNotificationPermissionController.requestPermissionIfNeeded(/* contextual= */ false)) {
             return true;
         }
 
