diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 6ae93362ad863ac54bedcc280129596a4100f778..fefea73c85b39f1cc3db9ceb653261493bcf0211 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -514,6 +514,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
