diff --git a/chrome/android/features/tab_ui/BUILD.gn b/chrome/android/features/tab_ui/BUILD.gn
index c46c9d51d592a2ae4ef364430bf878966a260043..c2f43ea43f290347ce88a5e2d283f3fe9ac9d872 100644
--- a/chrome/android/features/tab_ui/BUILD.gn
+++ b/chrome/android/features/tab_ui/BUILD.gn
@@ -104,6 +104,7 @@ android_resources("java_resources") {
     "java/res/xml/tab_archive_time_delta_preference.xml",
     "java/res/xml/tabs_settings.xml",
   ]
+  sources += brave_android_features_java_resources
   deps = [
     ":java_strings_grd",
     "//chrome/android:chrome_app_java_resources",
