diff --git a/components/sync/service/BUILD.gn b/components/sync/service/BUILD.gn
index 7c1d22c8b01876295ea1c294730c2ff9c9991545..a991ed650c526c51f69e10ea4c53cc6e0e5a2acc 100644
--- a/components/sync/service/BUILD.gn
+++ b/components/sync/service/BUILD.gn
@@ -124,4 +124,5 @@ static_library("service") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
