diff --git a/components/sync/service/BUILD.gn b/components/sync/service/BUILD.gn
index 3b4f497f4899a06c485a1c84876068f8dfc078ac..a1e5e1bb9b903acaf29a89cde177d2dadc37f115 100644
--- a/components/sync/service/BUILD.gn
+++ b/components/sync/service/BUILD.gn
@@ -125,4 +125,5 @@ static_library("service") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
