diff --git a/third_party/polymer/v3_0/BUILD.gn b/third_party/polymer/v3_0/BUILD.gn
index d2414dbe96c99817af1f1f23c25bd949d3309b7a..85dd0abbe2aa9de6851d0d4bbf9c60966db97e64 100644
--- a/third_party/polymer/v3_0/BUILD.gn
+++ b/third_party/polymer/v3_0/BUILD.gn
@@ -43,6 +43,7 @@ if (is_chromeos) {
 }
 
 if (is_chromeos_ash) {
+} if (!is_android) {
   js_files += [
     "iron-a11y-announcer/iron-a11y-announcer.js",
     "iron-behaviors/iron-control-state.js",
@@ -278,6 +279,7 @@ ts_library("library") {
   }
 
   if (is_chromeos_ash) {
+  } if (!is_android) {
     definitions += [
       "components-chromium/iron-a11y-announcer/iron-a11y-announcer.d.ts",
       "components-chromium/iron-collapse/iron-collapse.d.ts",
@@ -302,4 +304,5 @@ ts_library("library") {
       "components-chromium/paper-tooltip/paper-tooltip.d.ts",
     ]
   }
+  import("//brave/third_party/polymer/v3_0/sources.gni") visibility += brave_polymer_library_visibility
 }
