diff --git a/BUILD.gn b/BUILD.gn
index 383cdc89b3806db4c128d74ac3a6a4ab6d2c0aef..53371737adad5c85e29ffaf2e930add5cdd1b825 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -751,6 +751,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3151,6 +3152,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
