diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 86173a412d187585543a3137248d272f505e1816..83ff66a88681e3927dd7b56cc76bb991daa3a0f2 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3100,6 +3100,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
