diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 66dd3eac7b5a71a44d83b37fb2bc76147c29d617..eeb1b3e2e3da8f5407d5e43e88c0411a0beed852 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3102,6 +3102,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
