diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 32ce528ec78923b90061b841eead45d7e99a2f7b..15a198ad19176a714cab76d473291314ddfe9ab7 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3308,6 +3308,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
